TYPO3.Flow/Classes/TYPO3/Flow/Utility/Files.php

```diff
@@ -377,7 +378,11 @@ public static function unlink($pathAndFilename)
         try {
             // if not on Windows, call PHPs own unlink() function
             if (DIRECTORY_SEPARATOR === '/' || is_file($pathAndFilename)) {
-                return @\unlink($pathAndFilename);
+                if (!@\unlink($pathAndFilename)) {
```

Comment:
We even do check for the file existence in the line above (did not see that at first). I understand now that checking if the file exists and trying to fail silently are two different concerns. Thanks for explaining!